bouncer/bouncer.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM public.ecr.aws/sam/build-python3.9:latest
+
+## Initial setup
+
+COPY ./bouncer ${LAMBDA_TASK_ROOT}/
+COPY ./models.py ${LAMBDA_TASK_ROOT}/
+COPY ./requirements-bouncer.txt ${LAMBDA_TASK_ROOT}/
+
+RUN pip install --upgrade pip
```

Comment:
This should probably be done even before copying the requirements so that docker caches the results